### PR TITLE
Add support for ABI68

### PR DESCRIPTION
**Pull request by SveSop**
This should add support for SDK 7.6.0 albeit with possible issues regarding nvcuda.

### CODE REVIEW

File: src/nvoptix_68.c [hunk 1]
```diff
@@ -0,0 +1,452 @@
...
+
+static OptixResult __cdecl optixOpacityMicromapArrayComputeMemoryUsage_68(OptixDeviceContext context, const void *buildInput, void *bufferSizes)
+{
+    TRACE("(%p, %p, %p)\n", context, buildInput, bufferSizes);
+    return optixFunctionTable_68.optixOpacityMicromapArrayComputeMemoryUsage(context, buildInput, bufferSizes);
+}
+
+static OptixResult __cdecl optixOpacityMicromapArrayBuild_68(OptixDeviceContext context, CUstream stream, const void *buildInput, const void *buffers)
+{
+    TRACE("(%p, %p, %p, %p\n", context, stream, buildInput, buffers);
```
Comment (Saancreed) [inline src/nvoptix_68.c]:
```suggestion
    TRACE("(%p, %p, %p, %p)\n", context, stream, buildInput, buffers);
```

File: src/nvoptix_68.c [hunk 2]
```diff
@@ -0,0 +1,452 @@
...
+    return optixFunctionTable_68.optixOpacityMicromapArrayBuild(context, stream, buildInput, buffers);
+}
+
+static OptixResult __cdecl optixOpacityMicromapArrayGetRelocationInfo_68(OptixDeviceContext context, CUdeviceptr opacityMicromapArray, void *info)
+{
+     TRACE("(%p, %p, %p\n", context, opacityMicromapArray, info);
```
Comment (Saancreed) [inline src/nvoptix_68.c]:
```suggestion
     TRACE("(%p, %p, %p)\n", context, opacityMicromapArray, info);
```

File: src/nvoptix_68.c [hunk 3]
```diff
@@ -0,0 +1,452 @@
...
+     return optixFunctionTable_68.optixOpacityMicromapArrayGetRelocationInfo(context, opacityMicromapArray, info);
+}
+
+static OptixResult __cdecl optixOpacityMicromapArrayRelocate_68(OptixDeviceContext context, CUstream stream, const void *info, CUdeviceptr targetOpacityMicromapArray, size_t targetOpacityMicromapArraySizeInBytes)
+{
+     TRACE("(%p, %p, %p, %p, %zu\n", context, stream, info, targetOpacityMicromapArray, targetOpacityMicromapArraySizeInBytes);
```
Comment (Saancreed) [inline src/nvoptix_68.c]:
```suggestion
     TRACE("(%p, %p, %p, %p, %zu)\n", context, stream, info, targetOpacityMicromapArray, targetOpacityMicromapArraySizeInBytes);
```

File: src/nvoptix_68.c [hunk 4]
```diff
@@ -0,0 +1,452 @@
...
+OptixResult __cdecl optixQueryFunctionTable_68(
+    unsigned int numOptions,
+    int *optionKeys,
+    const void **optionValues,
+    void *functionTable,
+    size_t sizeOfTable)
+{
+    if (sizeOfTable != sizeof(OptixFunctionTable_68)) return OPTIX_ERROR_FUNCTION_TABLE_SIZE_MISMATCH;
+
+    OptixResult result = poptixQueryFunctionTable(60, numOptions, optionKeys, optionValues, &optixFunctionTable_68, sizeOfTable);
```
Comment (Saancreed) [inline src/nvoptix_68.c]:
```suggestion
    OptixResult result = poptixQueryFunctionTable(68, numOptions, optionKeys, optionValues, &optixFunctionTable_68, sizeOfTable);
```

**Review by Saancreed — COMMENTED** [review]
A few nits related to log message formats and one important thing in `poptixQueryFunctionTable`, otherwise looks good to me.

File: src/nvoptix_68.c (hunk 4 above)
Comment (SveSop) [reply src/nvoptix_68.c]: This was a critical miss by me tbh! Thanks for catching that one indeed 👍

**Pull request merged**